### PR TITLE
Resolvables

### PR DESCRIPTION
**Pull request by bandophahita**
Addresses #43

### CODE REVIEW

File: screenpy_selenium/resolutions/is_clickable.py [hunk 1]
```diff
@@ -4,25 +4,27 @@
...
-    matcher: IsClickableElement
-    line = "clickable"
-    matcher_function = is_clickable_element
+    def describe(self) -> str:
+        """Describe the Resolution's expectation."""
+        return "clickable"
 
-    def __init__(self) -> None:  # pylint: disable=useless-super-delegation
-        super().__init__()
+    @beat("... hoping it's clickable")
```
Comment (perrygoy) [inline screenpy_selenium/resolutions/is_clickable.py]: I think these need punctuation at the end, i.e.
```python
@beat("... hoping it's clickable.")
```

File: tests/test_resolutions.py [hunk 2]
```diff
@@ -223,11 +250,13 @@ def test_descriptions(self) -> None:
...
+        assert ip.describe() == "present"
+        _assert_descriptions(ip.resolve(), element, expected)
 
     def test_type_hint(self) -> None:
         ip = IsPresent()
-        annotation = ip.__annotations__["matcher"]
+        annotation = ip.resolve.__annotations__["return"]
         assert annotation == "IsPresentElement"
-        assert type(ip.matcher) == IsPresentElement
+        assert type(ip.resolve()) == IsPresentElement
```
Comment (perrygoy) [inline tests/test_resolutions.py:271]: Don't we have a `test_matches_protocols` test that we run? Maybe that's just for Actions?

Comment (perrygoy) [reply tests/test_resolutions.py:271]: Looks like we don't have those tests in `screenpy` for Resolutions, hm...

**Review by perrygoy — COMMENTED** [review]
I feel like i'm forgetting something, but i also had these two thoughts.

File: screenpy_selenium/resolutions/is_clickable.py (hunk 1 above)
Comment (bandophahita) [reply screenpy_selenium/resolutions/is_clickable.py]: updated.  I also somehow missed adding two tests for confirming these messages.

**Review by perrygoy — APPROVED** [review]
Nice work, @bandophahita! Finally got these to follow the new standard. 👍

**Pull request merged**